### PR TITLE
Preference update on sign up

### PR DESCRIPTION
**Pull request by aezouhri**
<!Pull Request Template>

## Description

<!Please provide a summary of the changes you have made. Include any relevant motivation and context. Mention any dependencies that are required for this change.>
Created the user dashboard. Currently have the preferences on the dashboard ensuring it is loaded properly and working properly.

## Tests
Tests cover the interaction of the buttons based of the data from mocked db objects.

## UI Changes
image

## Issues
#47 #66 

<!Mention any issues or bugs this PR addresses or fixes. Link any related issues.>

## Additional Notes
_Add any other information about the PR here._
Need to work putting the preferences on pop-up window on first sign up. Otherwise they will be located on the settings page.
---

### CODE REVIEW

**Comment by aezouhri** [comment]
Need help, some of the tests are not passing. Can someone assist ?

**Comment by saanbe16** [comment]
> Need help, some of the tests are not passing. Can someone assist ?

I'll look into this

**Review by aezouhri — COMMENTED** [review]
Thanks for looking into this!

File: web/src/components/Login.js [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
+        //toast.success('Login successful');
         // Store the token in localStorage
         localStorage.setItem('token', response.data.token);
-        // Store user information in session storage
-        sessionStorage.setItem('user', JSON.stringify(response.data.user));
-        window.postMessage({
-          type: "LOGIN_SUCCESS",
-          token: localStorage.getItem('token'),
-          user: sessionStorage.getItem('user')
-        }, "*");
```
Comment (saanbe16) [inline web/src/components/Login.js:28]: @aezouhri This code needs to be put back, in order for the extension login to work, please put it back and re push, If you put it somewhere else let me know where it is so i can double check that the logic works

**Review by saanbe16 — CHANGES_REQUESTED** [review]

**Review by saanbe16 — APPROVED** [review]
This is good to merge now

**Pull request merged**